<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
       <x>0</x>
       <y>0</y>
       <width>296</width>
       <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
       <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
               <widget class="QLabel" name="lbName">
                   <property name="text">
                       <string>Name</string>
                   </property>
               </widget>
           </item>
           <item row="1" column="0">
               <widget class="QLabel" name="label">
                   <property name="text">
                       <string>Customer</string>
                   </property>
               </widget>
           </item>
           <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                   <property name="text">
                       <string>Budget</string>
                   </property>
               </widget>
           </item>
           <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="sbBudget">
                   <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="buttonSymbols">
                       <enum>QAbstractSpinBox::NoButtons</enum>
                   </property>
                   <property name="prefix">
                       <string/>
                   </property>
                   <property name="maximum">
                       <double>999999999.000000000000000</double>
                   </property>
               </widget>
           </item>
           <item row="2" column="2">
               <spacer name="horizontalSpacer">
                   <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                       <size>
                           <width>40</width>
                           <height>20</height>
                       </size>
                   </property>
               </spacer>
           </item>
           <item row="2" column="3">
               <widget class="QLabel" name="label_3">
                   <property name="text">
                       <string>Time Budget</string>
                   </property>
               </widget>
           </item>
           <item row="2" column="4">
               <widget class="kemai::app::DurationEdit" name="leTimeBudget">
                   <property name="text">
                       <string>0</string>
                   </property>
                   <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
               </widget>
           </item>
           <item row="1" column="1" colspan="4">
               <widget class="QComboBox" name="cbCustomer"/>
           </item>
           <item row="0" column="1" colspan="4">
               <widget class="QLineEdit" name="leName">
                   <property name="maxLength">
                       <number>150</number>
                   </property>
               </widget>
           </item>
       </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
            <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
        </property>
    </widget>
   </item>
  </layout>
 </widget>
    <customwidgets>
        <customwidget>
            <class>kemai::app::DurationEdit</class>
            <extends>QLineEdit</extends>
            <header>durationedit.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>ProjectDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
